apiserver/facades/agent/uniter/register.go

```diff
@@ -54,6 +54,7 @@ func newUniterAPI(stdCtx context.Context, ctx facade.ModelContext) (*UniterAPI,
 		stdCtx, ctx,
 		Services{
 			ApplicationService:      domainServices.Application(),
+			ResolveService:          domainServices.Resolve(),
```

Comment:
I'm wondering if we'll end up being able to lasso some service methods such as from this one and from application that are isolated enough to constitute a `unitagent` domain or some such.